curator/run.sh

```diff
@@ -7,6 +7,7 @@ fi
 if [ "$CURATOR_LOG_LEVEL" = WARN ] ; then
     export CURATOR_LOG_LEVEL="WARNING"
 fi
+PYTHON_BIN=${PYTHON_BIN:-python3}
```

Comment:
does it normally run with python 3?

Reply:
There is a descrepency between the upstream and downstream images where I believe when I tested the scratch builds this was the only way to resolve